helm-charts/mock-classifier/templates/hpa.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "mock-classifier.fullname" . }}
```

Please add `-hpa` postfix so we can easily identify this.
```suggestion
  name: {{ include "mock-classifier.fullname" . }}-hpa
```